docs/datasets_and_tables/gtfs_schedule.md

```diff
@@ -1,15 +1,46 @@
-# GTFS Schedule (WIP)
+# GTFS Schedule
 
-## Data
+## How to view the `GTFS Schedule` documentation
 
-| dataset | description |
+Visit the [dbt Cal-ITP warehouse documentation](https://dbt-docs.calitp.org/#!/overview).
+
+For help navgating the documentation, visit [Navigating the dbt Docs](navigating-dbt-docs).
+
+## Warehouse Schemas
+
+### Source
+
+| dataset name | description |
+| ------- | ----------- |
+| `gtfs_schedule_type2` | Tables with GTFS-Static feeds across history (going back to April 15 2021). These are stored as type 2 slowly changing dimensions. They have `calitp_extracted_at` and `calitp_deleted_at` fields. |
+| `gtfs_schedule_history` | Data in the `gtfs_schedule_history` dataset in BigQuery |
+
+### Staging
+
+| dataset name | description |
 | ------- | ----------- |
-| gtfs_schedule | Latest warehouse data for GTFS Static feeds. See the [GTFS static reference](https://developers.google.com/transit/gtfs/reference). |
-| gtfs_schedule_type2 | Tables with GTFS Static feeds across history (going back to April 15th, 2021). These are stored as type 2 slowly changing dimensions. They have `calitp_extracted_at`, and `calitp_deleted_at` fields. |
-| (internal) gtfs_schedule_history | External tables with all new feed data across history. |
+| `gtfs_schedule` | Latest warehouse data for GTFS-Schedule feeds. |
+
+### See Also
+The [GTFS-Schedule Data Standard](https://developers.google.com/transit/gtfs/).
+
+## `dbt` Project Directories
+More information on the [navigating the docs from the project-perspective](navigating-dbt-docs)
+
+### Source tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_history` |
+| `gtfs_type2` |
 
-## Dashboards
+### Staging tables
+| Directory name |
+| ------- |
+| `gtfs_schedule_latest_only` |
```

Comment:
I would actually move these under "views", sorry I didn't catch before (this is the data we publish on the open data portal)

Reply:
no worries at all! let me know if my changes look right!